crowbar_framework/app/views/installer/upgrades/restore.haml

```diff
@@ -19,15 +19,22 @@
             %p
               = t(".description")
 
-            - @steps.each do |step|
-              .alert.alert-info{ step: step, style: "display:none;" }
-                = icon_tag "spin fa-spinner", t("installer.upgrades.restore.#{step}")
-
-            .alert.alert-danger{ style: "display:none;" }
-              = t(".failed")
-
-            .alert.alert-success{ style: "display:none;" }
-              = t(".success")
+            .upgrade{ data: { status_check: true } }
+              - @steps.each do |step|
+                .alert.alert-info.hidden{ data: { step: step } }
+                  = icon_tag "spin",
+                    t("installer.installers.show.#{step}"),
```

Comment:
```ActionView::Template::Error (translation missing: en.installer.installers.show.restore_crowbar):```

I assume the same is true for the other restore steps. Did you mean to say ```installer.upgrades.restore.#{step}``` here?